src/core/CCOpt.mli

```diff
@@ -94,6 +98,14 @@ val (<$>) : ('a -> 'b) -> 'a t -> 'b t
 
 (** {2 Alternatives} *)
 
+val or_ : else_:('a t) -> 'a t -> 'a t
+(** [or_ b a] is [a] if [a] is [Some _], [b] otherwise
+    @since NEXT_RELEASE *)
+
+val or_lazy : (unit -> 'a t) -> 'a t -> 'a t
```

Comment:
Yes indeed, but we must live with that. It is especially critical for `or_` because the two arguments are impossible to distinguish only with their type. My feeling is that `or_lazy` should be as close to `or_` as possible…